merlin/systems/dag/ops/faiss.py

```diff
@@ -95,14 +203,44 @@ def compute_input_schema(
     def compute_output_schema(
         self, input_schema: Schema, col_selector: ColumnSelector, prev_output_schema: Schema = None
     ) -> Schema:
+        """
+        Compute the input schema of this node given the root, parents and dependencies schemas of
```

Comment:
s/input/output/, correct?

Same req for serial comma: "...root, parents, and..."

ending punctuation, please.